UnityProject/Assets/Scripts/Objects/Machines/MassDriver.cs

```diff
@@ -147,7 +147,7 @@ private void ThrowItem(UniversalObjectPhysics item, Vector3 throwVector)
 			Vector3 vector = item.transform.rotation * throwVector;
 			UniversalObjectPhysics itemTransform = item.GetComponent<UniversalObjectPhysics>();
 			if (itemTransform == null) return;
-			itemTransform.NewtonianPush(vector, 1, inAim:BodyPartType.Chest ,inThrownBy : gameObject );
+			itemTransform.NewtonianPush(vector, 30, inAim:BodyPartType.Chest ,inThrownBy : gameObject );
```

Comment:
Might wanna increase lines 156 and 164 too?

Reply:
Sounds good